pinot-hadoop/pom.xml

```diff
@@ -77,6 +77,10 @@
           <groupId>org.apache.pinot</groupId>
```

Comment:
do we still need these exclusions? or did you change all the avro versions to a loewr version?

Reply:
There's no harm with these exclusions. Let's keep it here.
And we don't need to change all the avro version as well. The only required changes is to downgrade the newly added version of avro-mapred-hadoop2.

Reply:
do they have conflicting classes at all?

Reply:
No, we've tried our best to use the compatible method. ;)